packages/ui/components/ui/incompatable-browser-banner/get-compatability.ts

```diff
@@ -0,0 +1,62 @@
+import { UAParser, UAParserInstance, IDevice } from 'ua-parser-js';
+
+const parser: UAParserInstance = new UAParser();
+
+const CompatableBrowsers: Record<string, number> = {
+  // browser name: min version
+  Chrome: 119,
```

Comment:
Is there a specific reason why Chrome 119 was chosen as the minimum version?

Reply:
Can we add a code comment pointing to that json?